src/main/scala/fs2/package.scala

```diff
@@ -0,0 +1,39 @@
+package fs2
```

Comment:
weird, it's a new package, but we have an "internal" package already under src/main/scala/fs2/aws/internal